public/Day 32/index.html

```
@@ -1,11 +1,25 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8" />
+  <title>Social Media Dashboard | Analytics at a Glance</title>
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <title>Social Media Dashboard</title>
+  <meta name="description"
+    content="Monitor your social media growth and daily engagement with this sleek analytics dashboard.">
+  <!-- Open Graph / Facebook -->
+  <meta property="og:type" content="website">
```

HTMLHint (1.9.1)

[warning] The empty tag : [ meta ] must be self closed.

(tag-self-close)